### PR TITLE
Bug 1920367: When creating localvolumeset instance from the web console, the title for setting volumeMode is `Disk Mode`

### PR DESCRIPTION
**Pull request by rexagod**
Rename 'Disk Mode' to 'Volume Mode'.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1920367

### CODE REVIEW

**Comment by spadgett** [comment]
/override ci/prow/ceph-storage-plugin

The ceph-storage-plugin job should only have run on PRs with changes to `frontend/packages/ceph-storage-plugin`. It failed due to an unrelated CI config change. Overriding status on this PR.

File: frontend/packages/local-storage-operator-plugin/locales/ja/lso-plugin.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "Selecting all nodes will use the available disks that match the selected filters only on selected nodes.": "すべてのノードを選択すると、選択されたノードのみの選択されたフィルターに一致する利用可能なディスクが使用されます。",
   "Disk Type": "ディスクタイプ",
   "Advanced": "詳細",
-  "Disk Mode": "ディスクモード",
+  "Volume Mode": "ボリュームモード",
```
Comment (afreen23) [inline frontend/packages/local-storage-operator-plugin/locales/ja/lso-plugin.json:33]: We dont need to change in language-specific localization file

File: frontend/packages/local-storage-operator-plugin/locales/zh/lso-plugin.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
   "Selecting all nodes will use the available disks that match the selected filters only on selected nodes.": "选择所有节点将使用在所选节点上与所选过滤器相匹配的可用磁盘。",
   "Disk Type": "磁盘类型",
   "Advanced": "高级",
-  "Disk Mode": "磁盘模式",
+  "Volume Mode": "音量模式",
```
Comment (afreen23) [inline frontend/packages/local-storage-operator-plugin/locales/zh/lso-plugin.json:33]:
```suggestion
"Disk Mode": "音量模式",
```


We dont need to change in language-specific localization file

**Review by afreen23 — CHANGES_REQUESTED** [review]

**Comment by afreen23** [comment]
Please run `yarn i18n` inside your frontend folder to update the keys.

**Pull request merged**